pkg/stargate/affinity_test.go

```diff
@@ -68,19 +68,6 @@ func Test_computeNodeAffinityLabels(t *testing.T) {
 		assert.Equal(t, "value1", actual["label1"])
 		assert.Equal(t, "value2", actual["label2"])
 	})
-
-	t.Run("affinity on zone label", func(t *testing.T) {
-		//goland:noinspection GoDeprecation
-		actual := computeNodeAffinityLabels(&cassdcapi.CassandraDatacenter{
-			Spec: cassdcapi.CassandraDatacenterSpec{
-				Racks: []cassdcapi.Rack{{
-					Name: "rack1",
-					Zone: "zone1",
-				}},
-			}}, "rack1")
-		require.NotNil(t, actual)
-		assert.Equal(t, "zone1", actual[zoneLabel])
-	})
```

Comment:
question: Do we not need to test affinity labels anymore?

Reply:
affinity labels are still tested, and `NodeAffinityLabels` have now superceded the `Zone` field that was previously used for this, and has now been fully removed.